libs/core/src/lib/shellbar/user-menu/shellbar-user-menu.component.scss

```diff
@@ -0,0 +1,5 @@
+.fd-shellbar__icon {
```

Comment:
We need to add this class to fundamental-styles, can you open a PR there as well?

Reply:
It is the same "problem" that we have on the `ProductMenuComponent`, we have a specific use case for this `fd-icon` and we don't have an `fd-shellbar__icon` class on fundamental-styles

Reply:
I changed the fd-icon for the fd-menu-addon and I was able to remove the CSS file